ert3/storage/_storage.py

```diff
@@ -463,17 +551,28 @@ def get_ensemble_record(
 
     param_names = _get_experiment_parameters(workspace, experiment_name)
     if record_name in param_names:
```

Comment:
Could this not be simplified? It seems there is one case where a list is made and combined, otherwise always the ensemble record is made directly.